dockers/docker-sonic-mgmt/Dockerfile.j2

```diff
@@ -56,6 +56,7 @@ RUN pip install cffi==1.10.0 \
                 psutil \
                 pyasn1==0.1.9 \
                 pyfiglet \
+                lazy-object-proxy==1.6.0 \
```

Comment:
Can we upgrade pylint to fix it? So no need to install lazy-object-proxy explicitly.

Reply:
I have tried pylint 1.81~1.9.5, but failed.
seems pylint not specific lazy-object-proxy version.